new-backoffice/extending/development-flow/vite-package-setup.md

```diff
@@ -126,8 +131,26 @@ This example declares a Dashboard as part of your Package, using the Vite exampl
 
 {% endcode %}
 
+{% hint style="info" %}
+Umbraco needs the name of the element that will render as default when our dashboard loads. This is specified in the manifest as the `elementName`. Another approach would be to define your default element in the TS code.
+
+To do this, you should add `default` to your `MyElement` class in the `my-element.js` file like so 
+
+```ts
+export default class MyElement extends LitElement {
+```
+
+{% endhint %}
+
+### Testing your package
+
+In order to be able to test your package, you will need to run your site. Before you do this, you will need to include all the files in the `src` folder and the `umbraco-package.json` file into your project. We recommend you do this by adding a few lines to your csproj file like so:
```

Comment:
On the last sentence it mentions "We recommend you do this by adding a few lines to your csproj file like so:" however there is no code examples or so. Can you please look into it? Otherwise the rest looks good 😊 :#h5YR

Reply:
Oops. I decided to not leave an example of the csproj as it's too specific for every case and won't add a lot of value. I have removed the last sentence. I think the next paragraph about being careful when adding the folders using Visual Studio should be enough.